src/test/java/com/google/cloud/spanner/jdbc/JdbcArrayTest.java

```diff
@@ -203,6 +207,75 @@ public void testCreateArrayTypeName() throws SQLException {
       assertThat(rs.getTimestamp(2)).isEqualTo(new Timestamp(1000L));
       assertThat(rs.next()).isFalse();
     }
+
+    SingerInfo singerInfo =
+        SingerInfo.newBuilder().setSingerId(1).setNationality("Country1").build();
+    array =
+        JdbcArray.createArray(
+            "PROTO", new SingerInfo[] {singerInfo, SingerInfo.getDefaultInstance(), null});
+    assertThat(array.getBaseType()).isEqualTo(ProtoMessageType.VENDOR_TYPE_NUMBER);
```

Comment:
nit: I know that most existing assertions in the JDBC driver use `assertThat`, but the preferred way for new code is to use `assertEquals` and the other standard junit assertion methods. Would you mind changing that here?